# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../scribus/math.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The usual pow, ... builtin for x86 - but not for many others.

  - Rene Rebe <rene@exactcode.de>

--- ./scribus/scribusview.cpp.vanilla	2004-06-12 15:23:03.058045128 +0200
+++ ./scribus/scribusview.cpp	2004-06-12 15:23:18.707666024 +0200
@@ -25,6 +25,7 @@
 #endif
 
 #include "pageback.h"
+#include <math.h>
 #include <qcolor.h>
 #include <qfont.h>
 #include <qfontmetrics.h>
